index.html

```diff
@@ -13,36 +13,38 @@
 <p>&nbsp;</p>
 <p>All material is  covered by the <a href="http://creativecommons.org/licenses/by/3.0/">Creative Commons Attribution 3.0 Unported License</a> - you are free to share or adapt content as long as you credit the Open Microscopy Environment. The exception to this is the screenshots and videos which can only be used for non-commercial purposes.</p>
 <p>&nbsp;</p>
-<h2 style="text-align: center;">New in OMERO.insight 5.0</h2>
+<h2 style="text-align: center;">New Features in OMERO.insight 5.1</h2>
 
 <div class="line-block">
   <div class="line"><br /></div> <!-- end #line -->
 </div><!-- end #line-block -->
 
-<h2><a class="anchor" id="pwreset"></a><span class="bold">Reset Your Password</span></h2>
+<h2><a class="anchor" id="pwreset"></a><span class="bold">Key-value Pair Annotations</span></h2>
 
-<p>To reset your password, go to the OMERO.web login page.</p>
-<p>Click on the <span class="bold">Forgot your password?</span> link below the password text box to open the reset screen.</p>
-<p>Enter your OMERO user name and email address and click <span class="bold">Submit</span>.
-	<p>Your new password will be emailed to your OMERO email address.</p>
+<p>OMERO now supports a new type of annotation - Key-value Pairs.</p>
```

Comment:
key-value pairs? Not a fan of the starting caps when it is mid sentence, especially as you have 'pairs' lowercase on the next line and the link.